### ISSUE TITLE
Nx broken in Termux - Cannot find module '@nx/nx-android-arm64'

### ISSUE THREAD

**Issue by superdav42** [nrwl/nx#26634]
### Current Behavior

Running in android termux
```
npx create-nx-workspace@latest
```
Fails with
```
NX   Platform not supported

This platform (android-arm64) is currently not supported by Nx.
```

### Expected Behavior

Nx should be able to support android-arm64 / termux.
Nx should also stop relying on binary blobs but compile from source like author packages.

### GitHub Repo

https://github.com/nrwl/nx

### Steps to Reproduce

1. Install termux an Android device from f-droid
2. Install nodejs-lts with `pkg install nodejs-lts`
3. Run `create-nx-workspace@latest`
4. It no workie

### Nx Report

```shell
N/A
```

### Package Manager Version

10.7

### Operating System

- [ ] macOS
- [ ] Linux
- [ ] Windows
- [X] Other (Please specify)

### Additional Information

https://github.com/nrwl/nx/issues/19485

**Comment by FrozenPandaz** [comment]
This was fixed with #22870. :tada: 

Android is now supported again via Wasm. Wasm is still new though so let us know if you run into any issues.

**Comment by raifbrah** [comment]
> This was fixed with #22870. :tada: 
> 
> 
> 
> Android is now supported again via Wasm. Wasm is still new though so let us know if you run into any issues.

Please tell me more specifically how it works?
